### DISCUSSION TITLE
Backup calling button

### DISCUSSION THREAD

**Discussion by Batmanoulis** [kyujin-cho/pixel-volte-patch#245] [Carrier-specific Experiences]
Hello from Greece
In beta a14 we lost the backup calling feature.
So when we use data from sim 2 and they call us in sim 1 we lose Internet connection.
I see the option to enable it on pixel ims but nothing happens. Also it doesn't have the option to enable the toggle for this.
Pixel 7 latest beta 

Is it possible to have back?
Thank you

**Comment by fillwithjoy1** [comment]
I heard about Google removing this specifically in their Pixel devices, but it's a beta for now. It's possible that they are working on a better Backup Calling protocol, so just wait for a bit

**Reply by Batmanoulis** [reply]
I do. But Google always adds something and removes something. I don't understand why remove something very necessary..

**Reply by fillwithjoy1** [reply]
google ㄟ( ▔, ▔ )ㄏ

they are retarded at times, lets not forget the stupidity that was hangouts. could have had a monopoly

**Reply by fillwithjoy1** [reply]
Read #272 for how to get it working
